README.md

```diff
@@ -1 +1,36 @@
-# Review-portfolio
\ No newline at end of file
+# Portfolio
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> this project is the second project for the full stack web development class by microverse.
```

Comment:
I suggest modifying the project title and description. Those should be tailored to the current project "Review-Portfolio".